CRITICAL - 2022-01-09 00:04:53 --> number_format(): Passing null to parameter #1 ($num) of type float is deprecated
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(8192, 'number_format()...', 'C:\\vhost\\ci1\\sy...', 934)
#1 C:\vhost\ci1\system\Database\BaseConnection.php(934): number_format(NULL, 6)
#2 C:\vhost\ci1\system\Debug\Toolbar\Collectors\Database.php(111): CodeIgniter\Database\BaseConnection->getConnectDuration()
#3 C:\vhost\ci1\system\Debug\Toolbar\Collectors\BaseCollector.php(115): CodeIgniter\Debug\Toolbar\Collectors\Database->formatTimelineData()
#4 C:\vhost\ci1\system\Debug\Toolbar\Collectors\BaseCollector.php(231): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->timelineData()
#5 C:\vhost\ci1\system\Debug\Toolbar.php(90): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#6 C:\vhost\ci1\system\Debug\Toolbar.php(364): CodeIgniter\Debug\Toolbar->run(1641708293.2643, 0.1387, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\RedirectResponse))
#7 C:\vhost\ci1\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\RedirectResponse))
#8 C:\vhost\ci1\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\RedirectResponse), NULL)
#9 C:\vhost\ci1\system\CodeIgniter.php(427): CodeIgniter\Filters\Filters->run('auth/logout', 'after')
#10 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-01-09 00:05:09 --> number_format(): Passing null to parameter #1 ($num) of type float is deprecated
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(8192, 'number_format()...', 'C:\\vhost\\ci1\\sy...', 934)
#1 C:\vhost\ci1\system\Database\BaseConnection.php(934): number_format(NULL, 6)
#2 C:\vhost\ci1\system\Debug\Toolbar\Collectors\Database.php(111): CodeIgniter\Database\BaseConnection->getConnectDuration()
#3 C:\vhost\ci1\system\Debug\Toolbar\Collectors\BaseCollector.php(115): CodeIgniter\Debug\Toolbar\Collectors\Database->formatTimelineData()
#4 C:\vhost\ci1\system\Debug\Toolbar\Collectors\BaseCollector.php(231): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->timelineData()
#5 C:\vhost\ci1\system\Debug\Toolbar.php(90): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#6 C:\vhost\ci1\system\Debug\Toolbar.php(364): CodeIgniter\Debug\Toolbar->run(1641708309.6635, 0.1201, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#7 C:\vhost\ci1\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#8 C:\vhost\ci1\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#9 C:\vhost\ci1\system\CodeIgniter.php(427): CodeIgniter\Filters\Filters->run('auth', 'after')
#10 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-01-09 00:43:34 --> Invalid file: auth/check.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/check.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth/check', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(58): view('auth/check')
#3 C:\vhost\ci1\app\Controllers\Auth.php(38): App\Controllers\Auth->render_page('auth/check')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 00:45:27 --> Invalid file: auth/check.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/check.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth/check', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(59): view('auth/check')
#3 C:\vhost\ci1\app\Controllers\Auth.php(39): App\Controllers\Auth->render_page('auth/check')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 01:21:15 --> Invalid file: auth/auth/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/auth/login...')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth/auth/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth/auth/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(47): App\Controllers\Auth->render_page('auth/auth/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 01:21:46 --> Invalid file: auth/auth/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/auth/login...')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth/auth/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth/auth/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(47): App\Controllers\Auth->render_page('auth/auth/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 01:23:59 --> Invalid file: auth/auth/login/.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/auth/login...')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth/auth/login...', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(81): view('auth/auth/login...')
#3 C:\vhost\ci1\app\Controllers\Auth.php(48): App\Controllers\Auth->render_page('auth/auth/login...')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 05:02:30 --> Call to undefined function App\Controllers\getPost()
#0 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#1 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#2 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-01-09 05:36:37 --> Call to undefined function set_value()
#0 C:\vhost\ci1\system\View\View.php(214): include()
#1 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\vhost\ci1\app\Controllers\Auth2.php(48): view('auth/login')
#4 C:\vhost\ci1\app\Controllers\Auth2.php(31): App\Controllers\Auth2->renderView('login')
#5 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#6 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#7 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-01-09 05:41:37 --> Call to undefined function set_value()
#0 C:\vhost\ci1\system\View\View.php(214): include()
#1 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\vhost\ci1\app\Controllers\Auth2.php(48): view('auth/login')
#4 C:\vhost\ci1\app\Controllers\Auth2.php(31): App\Controllers\Auth2->renderView('login')
#5 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#6 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#7 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-01-09 05:53:01 --> Undefined variable $view
#0 C:\vhost\ci1\app\Controllers\Auth2.php(51): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 51)
#1 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('login')
#2 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#3 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#4 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-09 05:53:46 --> Call to undefined function set_value()
#0 C:\vhost\ci1\system\View\View.php(214): include()
#1 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\vhost\ci1\app\Controllers\Auth2.php(52): view('auth/login')
#4 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('login')
#5 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#6 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#7 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-01-09 08:06:04 --> Undefined variable $rules_arr
#0 C:\vhost\ci1\app\Libraries\ValidationRules.php(122): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 122)
#1 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Libraries\ValidationRules::getRules(NULL)
#2 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#3 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#4 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-09 17:38:10 --> Undefined variable $view
#0 C:\vhost\ci1\app\Controllers\Auth2.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 99)
#1 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('auth/login')
#2 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#3 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#4 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-09 17:39:03 --> view(): Argument #2 ($data) must be of type array, null given, called in C:\vhost\ci1\app\Controllers\Auth2.php on line 99
#0 C:\vhost\ci1\app\Controllers\Auth2.php(99): view('auth2/login', NULL)
#1 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('login')
#2 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#3 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#4 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-09 17:40:18 --> view(): Argument #2 ($data) must be of type array, null given, called in C:\vhost\ci1\app\Controllers\Auth2.php on line 99
#0 C:\vhost\ci1\app\Controllers\Auth2.php(99): view('auth/login', NULL)
#1 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('login')
#2 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#3 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#4 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-09 17:40:21 --> view(): Argument #2 ($data) must be of type array, null given, called in C:\vhost\ci1\app\Controllers\Auth2.php on line 99
#0 C:\vhost\ci1\app\Controllers\Auth2.php(99): view('auth/login', NULL)
#1 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('login')
#2 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#3 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#4 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-09 17:40:45 --> view(): Argument #2 ($data) must be of type array, null given, called in C:\vhost\ci1\app\Controllers\Auth2.php on line 100
#0 C:\vhost\ci1\app\Controllers\Auth2.php(100): view('auth/login', NULL)
#1 C:\vhost\ci1\app\Controllers\Auth2.php(34): App\Controllers\Auth2->renderView('login')
#2 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#3 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#4 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-09 17:45:13 --> view(): Argument #2 ($data) must be of type array, null given, called in C:\vhost\ci1\app\Controllers\Auth2.php on line 66
#0 C:\vhost\ci1\app\Controllers\Auth2.php(66): view('common/header', NULL)
#1 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('login')
#2 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#3 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#4 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-01-09 21:17:27 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 38)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Profile.php(41): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Profile.php(25): App\Controllers\Profile->render_page('pages/profile')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Profile->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profile))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:21:53 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 38)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(89): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(36): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:22:10 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 38)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(89): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(36): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:23:25 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(89): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(36): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:23:51 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(87): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(34): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:25:17 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(88): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(35): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:25:55 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth.php(79): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:25:55 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth.php(79): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:26:09 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth.php(79): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:26:11 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(88): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(35): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:26:12 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(88): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(35): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:26:53 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth.php(79): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:26:54 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth.php(79): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:26:55 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(88): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(35): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:49:57 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth.php(79): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:49:58 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(92): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:49:59 --> Undefined variable $user_info
#0 C:\vhost\ci1\app\Views\common\header.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\View\View.php(214): include('C:\\vhost\\ci1\\ap...')
#2 C:\vhost\ci1\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('common/header', Array, true)
#4 C:\vhost\ci1\app\Controllers\Auth2.php(92): view('common/header', Array)
#5 C:\vhost\ci1\app\Controllers\Auth2.php(33): App\Controllers\Auth2->renderView('login')
#6 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#7 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#8 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-01-09 21:52:15 --> Invalid file: auth2/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth2/login.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth2/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth2/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 21:58:51 --> Invalid file: auth2/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth2/login.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth2/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth2/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 21:58:51 --> Invalid file: auth2/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth2/login.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth2/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth2/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 21:58:56 --> Invalid file: auth2/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth2/login.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth2/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth2/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 21:59:00 --> Undefined variable $entry
#0 C:\vhost\ci1\app\Controllers\Auth2.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\vhost\\ci1\\ap...', 37)
#1 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth2->index()
#2 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth2))
#3 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-01-09 21:59:22 --> Invalid file: auth2/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth2/login.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth2/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth2/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 22:01:50 --> Invalid file: auth2/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth2/login.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth2/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth2/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-01-09 22:04:42 --> Invalid file: auth2/login.php
#0 C:\vhost\ci1\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth2/login.php')
#1 C:\vhost\ci1\system\Common.php(1097): CodeIgniter\View\View->render('auth2/login', Array, true)
#2 C:\vhost\ci1\app\Controllers\Auth.php(80): view('auth2/login')
#3 C:\vhost\ci1\app\Controllers\Auth.php(62): App\Controllers\Auth->render_page('auth2/login')
#4 C:\vhost\ci1\system\CodeIgniter.php(824): App\Controllers\Auth->index()
#5 C:\vhost\ci1\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\vhost\ci1\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\vhost\ci1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
